pagopa/io-app
# feat: [IA-362,IA-363,IA-364,IA-319] -- Carousel accessibility

## Short description
Fixed carousel accessibility. Voice over now reads the slides.

## List of changes proposed in this pull request
- [locales/en/index.yml,locales/it/index.yml]: added an indication to the correct place of the help button, for the first carousel slide.
- ts/components/LandingCardComponent.tsx: added accessibility support to LandingCardComponent.
- ts/screens/authentication/LandingScreen.tsx: added accessibility support to every carousel slide. The first one, describes the component as well.
- ts/components/HorizontalScroll.tsx: removed specific carousel accessibility description in this generic component.

## How to test
iOS:
Run app, VoiceOver, and verify if every slide of the carousel is read.
Android:
Run app, TalkBack, and verify if every slide of the carousel is read.
